modules/owncloud_web/pages/web_for_users.adoc

```diff
@@ -49,39 +49,47 @@ Under `Personal` in the left sidebar you have your private space where you can u
 
 Under `Shares` you can see all files or folders that have been shared with you. You can accept the share or decline. You can see who the owner of the share is and who else it has been shared with. Then there's the time or date when the content has been shared.
 
-A click on the three vertical dots opens an actions menu. Actions to be performed on a file or folder depend on the file type (video, text, etc.) and on the permissions granted by the sharer.
+A click on the three vertical dots opens an actions menu. Actions you can performed on a file or folder depend on the file type (video, text, etc.) and on the permissions granted by the sharer.
 
 === Spaces
 
 Spaces are organizational units to sort content and make it available to certain members.
 
 Under `Personal`, every user has a private space, while the `Spaces` area is intended for collaboration, i.e., sharing on a larger scale.
 
-In spaces, editors can do a little more and there's the additional role of manager.
+In apaces, editors can do a little more and there's the additional role of manager.
```

Comment:
sounds strange - maybe a typo?

Reply:
> sounds strange - maybe a typo?

Definitely. I think I wanted to change it to S